howto.md

```diff
@@ -0,0 +1,177 @@
+* If you now see a clock face, it means that your Pebble is still configured, maybe by the previous owner, maybe by you. If you used it before, now is the time to write down the watchfaces and apps you want to reinstall after this procedure is done.
+* If you see a clock face, press middle right button, go to *Settings*, then *System*, and choose *Factory Reset*. This might take a few minutes.
+* Get your phone and install the [Pebble app from the Play store](https://play.google.com/store/apps/details?id=com.getpebble.android.basalt). Accept all access. Then, open the app.
+* You'll be greeted by a *Pebble Account* page. Simply tap *Skip login* twice to continue.
+* Choose the type of Pebble you have.
+* The app starts looking for nearby Pebbles. It should shortly see your Pebble, as *Pebble `xxxx`*. This same string can be seen on the Pebble, so you know the phone sees the right watch. Tap it.
+* The watch asks to pair. Press the upper right button. Tap *Pair* on the phone.
+* If the Pebble has an old firmware, it will check for updates. Tap *Continue*. Keep the watch on the charging cable, and close to the phone. This will take a few minutes to complete, after that the watch will reboot.
+* In the Pebble app, choose the language of the menus, notifications and messages on the watch.
+* To get get notifications on your Pebble, tap *Open settings screen*. Enable *Pebble*. Tap *Allow*.
+* If you have a Pebble Time or newer: To enable health, tap *Enable*. Then, tap *Agree*.
+* If you have a Pebble Time or newer: Choose your settings for health, then tap *Continue*, then enter your info and tap *Complete*.
+* Choose a watchface. then tap the arrow in the upper right corner.
+* Timeline doesn't work, so simply tap the arrow in the upper right corner.
+* Don't grab apps yet, tap the arrow in the upper right corner.
+* You're done. Well, the Pebble part is done. Tap *done*.
+
+Part two: connect the Pebble app to Rebble Web Services.
+
+* On the phone, open a decent browser (like Chrome), and go to [auth.rebble.io](https://auth.rebble.io/).
+* You are greeted with a number of ways to sign in. Choose one and sign in. **Remember your sign in credentials!** You will return to the Rebble site, and are greeted with *You're all set!* Read the page, then tap the *get started now* link in the first paragraph.
+* You are redirected to [boot.rebble.io](https://boot.rebble.io/), and are greeted with *Here we go!* Read the page, then tap the *switch to Rebble* button. The browser asks if you want to open it with Pebble. Tap *Open*.
+* If it won't open with Pebble, you'll need to use another browser and restart from the first bullet of this part two.
+* The Pebble app opens. One of three things will then happen:
+* * A Rebble screen will appear. It may have an orange spinner above it; you can safely ignore this.
```

Comment:
did you intend the following here perhaps:

* parent
    * child
    * child
        * grandchild

this is achieved via::

```markdown
* parent
    * child
    * child
        * grandchild
```

?

Reply:
Thanks, learned something new today. :-)